src/common/snippets/src/pass/extract_reshapes_from_mha.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/pass/extract_reshapes_from_mha.hpp"
+
+#include <openvino/opsets/opset1.hpp>
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/util/binary_elementwise_arithmetic.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "snippets/itt.hpp"
+#include "snippets/snippets_isa.hpp"
+#include "snippets/pass/mha_tokenization.hpp"
+
```

Comment:
Minor: it seems like the following headers are not needed:
* #include "openvino/op/util/binary_elementwise_arithmetic.hpp"
* #include "snippets/snippets_isa.hpp"
If it is indeed so, can we remove them maybe?

Reply:
Good idea, done. Thanks!